src/interface/whitelist/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+
+const {
+    asyncHandler,
+    asyncHandlerArray,
+} = require('../../infra/asyncHandler');
+
+const whitelistAddress = require('./address');
+const getAddress = require('./getAddress');
+const { whitelistAdmin } = require('../middleware');
+
+router.post('/address', asyncHandler(whitelistAdmin), asyncHandlerArray(whitelistAddress));
+router.get('/address/:address', asyncHandler(getAddress));
```

Comment:
Better name for the get route `/check/:address`

Reply:
`/address/:address ` associates with that below comment.